help/en/releasenotes/current-draft-warnings.shtml

```diff
@@ -22,10 +22,15 @@
             <li></li>
             <li></li>
             <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
+            <li>dcc4pc multi-system update</li>
```

Comment:
Is this line intended here? You have almost the same text eight lines below (line 33).